README.rst

```diff
@@ -32,28 +32,26 @@ See `doc/install.txt`_ in the source code archive for general information. Excep
 
 .. _doc/install.txt: doc/install.txt
 
-Python 3 or later is needed.
+Python 3.5 or later is needed.
 
-The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``pip install git+https://github.com/linkchecker/linkchecker.git``. See `#359 <https://github.com/linkchecker/linkchecker/issues/359>`_.
+The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``sudo pip3 install git+https://github.com/linkchecker/linkchecker.git``. See `#359 <https://github.com/linkchecker/linkchecker/issues/359>`_.
```

Comment:
I cannot in good conscience recommend `sudo pip3`.

Quoting [The Official Tutorial](https://packaging.python.org/tutorials/installing-packages/):

> Do _not_ run any of the commands in this tutorial with `sudo`: if you get a permissions error, come back to the section on creating virtual environments, set one up, and then continue with the tutorial as written.

Reply:
We could say ``pip3 install --user`` and link to:
https://packaging.python.org/tutorials/installing-packages/#installing-to-the-user-site

Reply:
I'll take sudo out for now and merge this, to not hold up this PR any longer for one point. We can always add more later.

Reply:
i would assume `--user` is the default if not `sudo`. also, `pip3` might be a Debianism, so we might want to be careful about that as well - some platforms point `pip` to `python3`, iirc. but that's another discussion i guess...

Reply:
Gentoo has `pip3`, as does Python from the Microsoft Store. I think we are OK.